### PR TITLE
fix: location of roots.exe in devmode

### LINKED ISSUE #4653: Faiure to extract CA on windows in dev mode
### Bug description

When starting podman desktop with `yarn watch` on windows it shows error in console:

```
Error while retrieving Windows certificates Error: spawnSync C:\p\podman-desktop\packages\main\dist\roots.exe ENOENT
    at Object.spawnSync (node:internal/child_process:1110:20)
    at spawnSync (node:child_process:897:24)
    at Object.execFileSync (node:child_process:940:15)
    at Object.func [as execFileSync] (node:electron/js2c/asar_bundle:2:2131)
    at Object.run (C:\p\podman-desktop\packages\main\dist\index.cjs:97425:30)
    at api2 (C:\p\podman-desktop\packages\main\dist\index.cjs:97932:12)
    at C:\p\podman-desktop\packages\main\dist\index.cjs:98055:7
    at new Promise (<anonymous>)
    at Certificates.retrieveWindowsCertificates (C:\p\podman-desktop\packages\main\dist\index.cjs:98053:28)
    at Certificates.retrieveCertificates (C:\p\podman-desktop\packages\main\dist\index.cjs:98035:19)
```

### Operating system

Windows 10 Pro

### Installation Method

Other

### Version

next (development version)

### Steps to reproduce

Run podman desktop from github repository root folder 

```yarn watch``

image

### Relevant log output

_No response_

### Additional context

_No response_

### PR DESCRIPTION
**Pull request by dgolovin**
### What does this PR do?

fixes exception when retrieving CA on windows.

### Screenshot/screencast of this PR

image

### What issues does this PR fix or reference?

fix #4653.

### CODE REVIEW

File: packages/main/src/plugin/certificates.ts [hunk 1]
```diff
@@ -81,6 +81,8 @@ export class Certificates {
       if (import.meta.env.PROD) {
         const rootExePath = path.join(process.resourcesPath, 'win-ca', 'roots.exe');
         wincaAPI.exe(rootExePath);
+      } else {
+        wincaAPI.exe(path.resolve(__dirname, '..', '..', '..', 'node_modules', 'win-ca', 'lib', 'roots.exe'));
```
Comment (jeffmaury) [inline packages/main/src/plugin/certificates.ts]:
```suggestion
        wincaAPI.exe(require.resolve('win-ca/lib/roots.exe'));
```

**Review by lstocchi — COMMENTED** [review]
Thanks Denis, this is an error i always saw but never had the time to look at. 🚀 
LGTM, with Jeff suggestion it would be even better

File: packages/main/src/plugin/certificates.ts (hunk 1 above)
Comment (dgolovin) [reply packages/main/src/plugin/certificates.ts]: verified on windows and it is the better way to resolve path to roots.exe.

**Review by jeffmaury — APPROVED** [review]
LGTM

**Review by lstocchi — APPROVED** [review]

**Pull request merged**